community/index.md

```diff
@@ -8,6 +8,7 @@ title: Get in touch!
     <p>We'd love to have your feedback, suggestions, and will be happy to help with anything you need. Just get in touch!</p>
 
     <ul>
+      <li><a href="https://lists.apache.org/list.html?jclouds.apache.org">Browse and participate</a> in our mailing lists</li>
```

Comment:
> Makes sense?

The explanation makes sense, but I wonder if that would encourage people to simply use the dev list for everything? In general, I wonder whether "default to user@" would be a better guide in terms of where to ask questions first?

@andreaturli @andrewgaul Would be curious to hear your thoughts on that?